extension/background/entityTypes.js

```diff
@@ -21,9 +21,19 @@ for (const id in metadata.music) {
   }
 }
 
+export const namedPages = [];
+
+for (const id in metadata.pageNames) {
+  const item = metadata.pageNames[id];
+  for (const name of item.names) {
+    namedPages.push(name);
+  }
+}
+
```

Comment:
I understand I need to create a `new Alternative` here can I get a bit more clarity on that @ianb